DESCRIPTION

```diff
@@ -1,7 +1,6 @@
 Package: camtrapdp
 Title: Read and Manipulate Camera Trap Data Packages
-Version: 0.2.1.9000
-Date: 2024-06-05
+Version: 0.2.2.9000
```

Comment:
Please revert to 0.2.1.9000. The progression in version numbers is:

- 0.2.1 (latest CRAN release)
- 0.2.1.9000 (current version number + 9000): ongoing work
- whatever version we decide to release next (major, minor, patch)